csharp/.vscode/extensions.json

```diff
@@ -1,9 +1,7 @@
 {
     "recommendations": [
-        "formulahendry.dotnet-test-explorer",
         "gaoshan0621.csharp-format-usings",
         "github.vscode-codeql",
-        "hbenl.vscode-test-explorer",
```

Comment:
According to my VS Code the CodeQL plugin depends on this.

Reply:
It does, but I think that will soon change.